Documentation/data-collection.md

```diff
@@ -648,6 +648,16 @@ data:
     #
     # cluster:velero_restore_total:max measures the current number of existent restores
     - '{__name__="cluster:velero_restore_total:max"}'
+    #
+    # owners: (@openshift/image-registry)
```

Comment:
If it's a GitHub team, it should be @openshift/openshift-team-image-registry.

Reply:
fixed